web/src/main/java/com/bazaarvoice/emodb/web/migrator/MigratorRateLimiter.java

```diff
@@ -0,0 +1,7 @@
+package com.bazaarvoice.emodb.web.migrator;
+
+public interface MigratorRateLimiter {
+    int getMaxWritesPerSecond(String migrationId);
+
+    void setMaxWritesPerSecond(String migrationId, int maxWritesPerSecond);
```

Comment:
Just a thought: `setMaxWritesPerSecond` is only called by `DeltaMigrator`, which gets the actual `StatusDAO` injected.  So it isn't really necessary for this API.  Plus it makes clear that anywhere it's injected is intended as a read-only user.